sklearn/metrics/__init__.py

```diff
@@ -97,6 +98,7 @@
     'precision_recall_fscore_support',
     'precision_score',
     'r2_score',
+    'label_ranking_loss',
```

Comment:
Could you instead add this line below `'label_ranking_average_precision_score'`?